cadc-download-manager-server/src/main/java/ca/nrc/cadc/dlm/server/DispatcherServlet.java

```diff
@@ -350,4 +320,138 @@ public Object run()
         }
     }
 
+
+    private void handleDeprecatedAPI(HttpServletRequest request) {
+
+        String uris = (String) request.getAttribute("uris");
```

Comment:
Moved this into ServerUtil & have it called inside getURIs, in case a uri list isn't found there. 
I opted off having it return a List because it also handles deprecated params input as well, placing results of that in the request object as an attribute. 

So it was either have 2 functions: one to handle URIs passed in returning a List, another to handle params & return those, or one that would alter both and set request attributes. 

(I see I've missed the WARN message - still have to add that.) 

Getting rid of the 'uris' string usage propagated into all the files, of course. (so far more added to the pull request after this.) 

Use of DownloadUtils.encode/decodeListURI should be eliminated now.

Reply:
added warning messages now. 

The only time deprecated parameters are checked for is if there are no normal parameters. (I would have thought?)

Reply:
Well, while diagnosing a page that was having DLM issues recently, I saw that the dev put in both uri and fileid params and set the value of the latter dynamically (with complete ad:{archive}/{filename} URIs!)... I don't know if the empty uri params get submitted or not, but it's clear that pages out there could be doing all manner of strange things or combining supported and deprecated params.